doc/usage.rst

```diff
@@ -23,27 +23,6 @@ extension. Commands on Fedora to install dependencies:
 * Python 3: ``sudo dnf install python3-devel``
 * PyPy: ``sudo dnf install pypy-devel``
 
-Windows notes
--------------
-
-On Windows, to allow pyperformance to build dependencies from source
-like ``greenlet``, ``dulwich`` or ``psutil``, if you want to use a
-``python.exe`` built from source, you should not use the ``python.exe``
-directly. Instead, you must run the little-known command ``PC\layout``
-to create a filesystem layout that resembles an installed Python::
-
-    .\python.bat -m PC.layout --preset-default --copy installed -v
-
-(Use the ``--help`` flag for more info about ``PC\layout``.)
-
-Now you can use the "installed" Python executable::
-
-    installed\python.exe -m pip install pyperformance
-    installed\python.exe -m pyperformance run ...
-
-Using an *actually* installed Python executable (e.g. via ``py``)
-works fine too.
-
```

Comment:
I was going with the assumption that using a source build *on Windows* is such a rare occurrence that this would be in the realm of CPython developers already operating out of a GH clone.  This would imply an up to date (-ish) checkout, therefore containing the bug fix.

Reply:
Yeah, that makes sense.  The catch is that we run benchmarks against older CPython versions too.

Reply:
I'll admit, however, I have no idea how any current workflows using pyperformance *on Windows* are implemented.  Although the current guidance seems to be quite heavy already.

Reply:
Very few have managed to successfully run PyPerformance on Windows, because it has been so broken for so long (and the test suite is still broken -- as soon as you start it, the process detaches and runs in the background, which I didn't even know was possible in ~DOS~ Windows :-).

I have a reason for building past versions from source though: You never know which aspects of your current setup (compiler version, Windows version, hardware, version of other tooling installed) might affect the performance of the binary. It wouldn't be the first time that someone reported a significant but mysterious/irreproducible speedup or slowdown between two releases only to find out that they were comparing binaries built with different compiler versions. If the goal is to measure the effect of the source code changes rather than simply the progress in compiler technology, it's important to try to keep as many variables unchanged as possible.